### ISSUE TITLE
[Sourcing][Accessibility][High][Jul] Navigable rows issue

### ISSUE THREAD

**Issue by I543348** [SAP/fundamental-ngx#10003]
#### Is this a bug, enhancement, or feature request?
Bug

#### Which versions of Angular and Fundamental Library for Angular are affected? (If this is a feature request, use current version.)
All these were tested as on 31st May 2023 on https://sap.github.io/fundamental-ngx/#/platform/table/ platform table component samples

#### If this is a bug, please provide steps for reproducing it.
**Navigable rows related issues:
Issue 7:** 
User cannot use arrow keys to navigate within the table rows when the whole row has the keyboard focus.
<img width="452" alt="image" src="https://github.com/SAP/fundamental-ngx/assets/88182358/b8e0ff89-6ae9-423a-8d7e-0280a72e8cc5">

#### Please provide relevant source code if applicable.
All these were tested as on 31st May 2023 on https://sap.github.io/fundamental-ngx/#/platform/table/ platform table component samples

**Comment by fkolar** [comment]
It works for me in both examples. `Activable and Navigatable rows` Once having focus on whole row, hit ENTER and individual cell is highlighted. 

<img width="1234" alt="Screenshot 2023-06-26 at 12 38 09" src="https://github.com/SAP/fundamental-ngx/assets/17149942/53a25fd6-1185-4fc0-b249-eb233ba328e8">
<img width="1304" alt="Screenshot 2023-06-26 at 12 38 05" src="https://github.com/SAP/fundamental-ngx/assets/17149942/d0ae4f70-5dd3-4f58-b8f2-dbdc17770f1a">


<img width="928" alt="Screenshot 2023-06-26 at 12 38 40" src="https://github.com/SAP/fundamental-ngx/assets/17149942/b2400ac0-f0c2-4544-b4d3-a07fa364e96d">
<img width="1340" alt="Screenshot 2023-06-26 at 12 38 36" src="https://github.com/SAP/fundamental-ngx/assets/17149942/4d1244a9-f966-413c-8195-ece461fb009d">


Closing. Feel free to Reopen in case I missed something. But based on the provided information functionality works as expected.

**Comment by I543348** [comment]
works fine

**Comment by sagar7259** [comment]
Hello,
This issue is verified and fixed, hence this ticket can be closed.

Regards,
Sagar(ATL Team)
